src/components/questions/__tests__/QuestionForm.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { describe, it, expect, vi, beforeEach } from "vitest";
+import { mount } from "@vue/test-utils";
+import QuestionForm from "../QuestionForm.vue"
+import EasyModal from "../../common/EasyModal.vue";
+import { createTestingPinia } from "@pinia/testing";
+import { useQuestionStore } from "../../../stores/questions";
+import { nextTick } from "vue";
+
+describe("QuestionForm.vue", () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it("v-model should set values to question's initState", async () => {
+    const wrapper = mount(QuestionForm, {
+      global: {
+        components: {
+          EasyModal,
+        },
+      },
+    });
+    wrapper.find("#point").setValue(1);
+    wrapper.find("#text").setValue("question");
+    wrapper.find("#category").setValue("React");
+    wrapper.find("#answer").setValue("answer");
+    expect(wrapper.vm.questionData).toMatchObject({
+      point: 1,
+      text: "question",
+      id: 0,
+      category: "React",
+      answer: "answer",
+    });
+  });
+
+  it("should post question", async () => {
+    const wrapper = mount(QuestionForm, {
```

Comment:
you can create a function before the suit like
```
const getWrapper = () => mount(QuestionForm, {
      global: {
        components: {
          EasyModal,
        },
        plugins: [createTestingPinia()],
      },
    });
```
and use
`const wrapper = getWrapper()` in each test